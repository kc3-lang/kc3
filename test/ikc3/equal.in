a = 1
b = 1
c = 2
[d, e] = [1, 2]
