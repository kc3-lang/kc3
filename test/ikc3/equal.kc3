quote a = 1
a = 1
quote b = 1
b = 1
quote c = 2
c = 2
quote [d, e] = [1, 2]
[d, e] = [1, 2]
quote {1, 2} = {f, g}
{1, 2} = {f, g}
quote ? = 1
? = 1
quote to_lisp(quote a = 1 && b = 2)
to_lisp(quote a = 1 && b = 2)
quote if (a = 1 && b = 2) do {a, b} end
if (a = 1 && b = 2) do {a, b} end
