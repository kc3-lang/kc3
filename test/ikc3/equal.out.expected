a = 1
1
b = 1
1
c = 2
2
[d, e] = [1, 2]
[1, 2]
{1, 2} = {f, g}
{1, 2}
? = 1
1
